addons/a11y/src/components/Report/__snapshots__/HighlightToggle.test.js.snap

```diff
@@ -37,8 +38,10 @@ exports[`HighlightToggle component should match snapshot 1`] = `
             "HTML_TAGNAME_TEXT_TRANSFORM": "lowercase",
             "HTML_TAG_COLOR": "rgb(168, 148, 166)",
             "OBJECT_NAME_COLOR": "rgb(136, 19, 145)",
+            "OBJECT_PREVIEW_ARRAY_MAX_PROPERTIES": 10,
+            "OBJECT_PREVIEW_OBJECT_MAX_PROPERTIES": 5,
             "OBJECT_VALUE_BOOLEAN_COLOR": "rgb(28, 0, 207)",
-            "OBJECT_VALUE_FUNCTION_KEYWORD_COLOR": "rgb(170, 13, 145)",
+            "OBJECT_VALUE_FUNCTION_PREFIX_COLOR": "rgb(13, 34, 170)",
```

Comment:
are these changes the result of something else?

Reply:
I don't really know TBH, I just updated the snapshots